src/phoenix/experimental/evals/models/rate_limiters.py

```diff
@@ -6,14 +6,15 @@
 
 from typing_extensions import ParamSpec
 
+from phoenix.exceptions import PhoenixException
 from phoenix.utilities.logging import printif
 
 ParameterSpec = ParamSpec("ParameterSpec")
 GenericType = TypeVar("GenericType")
 AsyncCallable = Callable[ParameterSpec, Coroutine[Any, Any, GenericType]]
 
 
-class UnavailableTokensError(Exception):
+class UnavailableTokensError(PhoenixException):
     pass
```

Comment:
Does sub-classing here prevent retries on unavailable token errors? Is that what we want?